File: public/js/src/models/entities/Assessment.js
//Класс Ассесмент
export class Assessment{
    constructor(ID, name, date, disposer, candidates, employees, state){
        this.ID = ID;                   //ID
        this.name = name;               //Название
        this.date = date;               //Дата и время
        this.disposer = disposer;       //Распорядитель
        this.candidates = candidates;   //Кандидаты приглашённые на ассессмент
        this.employees = employees;     //Сотрудники присутствующие на ассессменте
        this.state = state;             //Состояние ассессмента
    }
};

File: app/models/entities/Assessment.go
package entities

//Assessment структура сущности ассессмента
type Assessment struct {
	ID         int64       `json:"ID"`       //Идентификатор
	State      string      `json:"state"`    //Вненшний ключ состояния
	Disposer   int64       `json:"disposer"` //Внешний ключ распорядителя
	Title      string      `json:"title"`    //Название
	Date       string      `json:"date"`     //Дата
	Employees  []Employee  `json:"-"`        //Сотрудники
	Candidates []Candidate `json:"-"`        //Кандидаты
}
